Plugin/Model/MenuBuilder.php

```diff
@@ -15,21 +15,12 @@ final class MenuBuilder
 {
     public function afterGetResult(Builder $subject, Menu $result): Menu
     {
-        $shouldRemoveMenu = false;
```

Comment:
This code was in place to remove the "Extensions" menu item if there is not child items under it. 

After the changes in pull request to this code, broke this behaviour. May you please give it a check.

Reply:
I can't reproduce the issue, please could you give some inputs?
If you look the code, the menu is removed if there is no items/no allowed items.

Reply:
@rav-redchamps was you able to reproduce the error?

Reply:
Hey @thomas-kl1 

Very sorry for long delay on this. I will be checking this today/tomorrow & get back to you.